### DISCUSSION TITLE
CoD крашит

### DISCUSSION THREAD

**Discussion by FRUDii** [bol-van/zapret#847] [Q&A]
С запущеным zapretom крашит call of duty, есть варианты решения проблемы?

**Comment by numbereleven-a** [comment]
только если на роутере развернуть
https://github.com/bol-van/zapret/discussions/755#discussioncomment-11198786

**Comment by 2lJuggerl22** [comment]
Такая же тема была, с помощью процесс хакера закрываешь дерево процессов запрета, в цмд пишешь sc query windivert и потом sc stop windivert, для меня все работало и колда запускалась, к слову если снова запрет запустить колда вылетит это я уже незнаю как фиксить

**Comment by EgorWeders** [comment]
Это корректное поведение античит системы (если вы конечно не про cod 4 или тому подобное). Вас могут еще и забанить за это. Причем если развернете на роутере риск бана останется.
